CONTRIBUTING.md

```diff
@@ -66,3 +66,12 @@ See the [RELEASING.md](RELEASING.md) for instructions.
 Please note that libraries need bounds on the version of their dependencies to avoid bitrot and be effectively reusable.
 
 [CHaP]: https://github.com/input-output-hk/cardano-haskell-packages
+
+## Improving `git blame`
+
+You can have `git blame` ignore the reformatting done when [introducting fourmolu](https://github.com/IntersectMBO/cardano-cli/pull/835/commits/9d1fd093071278be77428dba189d958d5b7a7aeb) by setting this in your `.git/config` file:
```

Comment:
I think this should be set on a per repo basis - not every repo has .git-blame-ignore-revs and if it doesn't, it breaks `git blame`:
```
$ git blame flake.nix
fatal: could not open object name list: .git-blame-ignore-revs

```
I think it would be better to recommend this command `git config --local blame.ignoreRevsFile .git-blame-ignore-revs`

Reply:
@carbolymer> yep I meant the repository's local git config file (i.e. `.git/config`, as opposed to the global `.gitconfig`), but anyway, it's better to give the command than the snippet; so I changed :+1: